client/src/components/Checkout/Checkout.css

```diff
@@ -43,6 +44,10 @@
     color: white;
 }
 
-/* @media only screen and (min-width : 1200px) {
-    .container { max-width: 1500px; }
-} */
\ No newline at end of file
+.radio-selection {
+    text-align: left;
+}
+
+input[type="radio"] {
```

Comment:
Please don't use input instead use specific class names as it overrides all input tags this way.